src/java.xml/share/classes/module-info.java

```diff
@@ -27,6 +27,231 @@
  * Defines the Java API for XML Processing (JAXP), the Streaming API for XML (StAX),
  * the Simple API for XML (SAX), and the W3C Document Object Model (DOM) API.
  *
+ * @implNote
+ * <h2>Implementation Specific Features and Properties</h2>
+ *
+ * In addition to the standard features and properties described within the public
+ * APIs of this module, the JDK implementation supports a further number of
+ * implementation specific features and properties. This section describes the
+ * naming convention, System Properties, jaxp.properties, scope and order,
+ * and processors to which a property applies. A table listing the implementation
+ * specific features and properties which the implementation currently supports
+ * can be found at the end of this note.
+ *
+ * <h3>Naming Convention</h3>
+ * The names of the features and properties are fully qualified, composed of a
+ * prefix and name.
+ * <p>
+ * <h4>Prefix</h4>
+ * The prefix for JDK properties is defined as:
+ * <pre>
+ *     {@code http://www.oracle.com/xml/jaxp/properties/}
+ * </pre>
+ *
+ * The prefix for features:
+ * <pre>
+ *     {@code http://www.oracle.com/xml/jaxp/features/}
+ * </pre>
+ *
+ * The prefix for System Properties:
+ * <pre>
+ *     {@systemProperty jdk.xml.}
```

Comment:
The use of {@systemProperty...} seems inappropriate, it should be used to refer to specific properties. (IMHO)

Reply:
will change to @code.